### DISCUSSION TITLE
More information on where Pypsa-Earth get its data from

### DISCUSSION THREAD

**Discussion by Constable1819** [pypsa-meets-earth/pypsa-earth#1100] [Q&A] [answered]
Halo, 

Where does pypsa-earth get its raw data from? Eg the hourly-demand of a certain country eg Angola in 2011. Is there any documentation or websites where the raw data has been collected from? I read this pdf 'PyPSA-Earth. A new global open energy system optimization model demonstrated in Africa' and saw that the 2030 and 2050 data is forecasted using the GEGIS package. 

This is the exact paragraph. But there is no mention of demands of 2011,2013,18, it says weather years instead.

'The model currently provides globally hourly demand predictions considering ’Shared Socioeconomic Pathways’ [51] scenarios for 2030, 2040, 2050 and 2100 and weather years of 2011, 2013 and 2018. The demand time series is created using the new contributed synde package [52], which implements a workflow management system to extract the demand data created with the open-source Global-Energy GIS (GEGIS) package [29].'

Please help.

**Comment by ekatef** [comment, marked as answer]
Hello @Constable1819! Thanks for the question. That is in fact a bit tricky part.

The demand is calculated considering weather conditions from the past (2011, 2013, 2018) but accounts for the trends linked with the socio-economic processes assuming SSP2-2.6 scenario in the future years (2030, 2040, 2050, 2100). [The model documentation](https://pypsa-earth.readthedocs.io/en/latest/data_workflow.html#pre-calculated-datasets) provides a bit more details on that.
